privacy-policy.md

```diff
@@ -0,0 +1,210 @@
+
+In some regions (like the EEA, UK, and Switzerland), you have certain rights under applicable data protection laws. These may include the right (i) to request access and obtain a copy of your personal information, (ii) to request rectification or erasure; (iii) to restrict the processing of your personal information; (iv) if applicable, to data portability; and (v) not to be subject to automated decision-making. In certain circumstances, you may also have the right to object to the processing of your personal information. You can make such a request by contacting us by using the contact details provided in the [How Can You Contact Us About This Notice?](#9-how-can-you-contact-us-about-this-notice) section below.
+
+We will consider and act upon any request in accordance with applicable data protection laws.
+ 
+If you are located in the EEA or UK and you believe we are unlawfully processing your personal information, you also have the right to complain to your Member State data protection authority or UK data protection authority.
+
+If you are located in Switzerland, you may contact the Federal Data Protection and Information Commissioner.
+
+#### Withdrawing your consent
+If we are relying on your consent to process your personal information, you have the right to withdraw your consent at any time. You can withdraw your consent at any time by contacting us by using the contact details provided in the [How Can You Contact Us About This Notice?](#9-how-can-you-contact-us-about-this-notice) section or updating your preferences.
+
+However, please note that this will not affect the lawfulness of the processing before its withdrawal nor will it affect the processing of your personal information conducted in reliance on lawful processing grounds other than consent.
+
+#### Opting out of marketing and promotional communications
+You can unsubscribe from our marketing and promotional communications at any time by Leave our Discord, Mute the channel, or by contacting us using the details provided in the [How Can You Contact Us About This Notice?](#9-how-can-you-contact-us-about-this-notice) section below. You will then be removed from the marketing lists. However, we may still communicate with you — for example, to send you service-related messages that are necessary for the administration and use of your account, to respond to service requests, or for other non-marketing purposes.
+
+#### Account Information
+
+If you would at any time like to review or change the information in your account or terminate your account, you can:
+ * Contact us
+ * Request removal of all or specific content
+ * Change your information through our websites
+ * Leave our services such as our Discord server
+
+Upon your request to terminate your account, we will deactivate or delete your account and information from our active databases unless it’s for legal reasons or it’s to prevent harm to others or yourself.
```

Comment:
We don't adminster accounts for anybody as-per-se - if this is wrt. the voting app, significant clarification of that and how that works is needed here and throughout this document.